db/migrate/20220527201132_add_author_and_company_to_articles.rb

```diff
@@ -0,0 +1,8 @@
+class AddAuthorAndCompanyToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :articles, :user, default: User.where(role: :superadmin).first.id, null: false
```

Comment:
I think it is strange to set default user id on the database level. If SuperAdmin id will change, You will need to create new migration. Moreover, in you schema you can see id: 1 as default value. So if i will try to setup application on my own computer, there will not this user when I will load schema, so, you need to setup this value on app level if it is needed.

Reply:
I understood my mistake, perhaps. I should set default value for existing records by executing command after migration. Am I right?

Reply:
I think this value should not be set on database level. As I think, if you need to set this value for previous values in database, you should make migration with null: true. Then you need update old fields by deploy script. And after that you need to create new migration taht will change constraint to null: false